network-api/networkapi/buyersguide/templates/buyersguide_home.html

```diff
@@ -60,6 +60,15 @@ <h1 class="h1-heading text-center mt-4">Be Smart. Shop Safe.</h1>
           >
         {% endif %}
 
+        {% if product.adult_content is True %}
```

Comment:
Is this the same code as in the category page template? Because then we can probably put it in its own fragment and just include that here.